src/app/v2/trade/[primary]/[numeraire]/page.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
+
+import { PairSelector } from '@/components/PairSelector';
+import { Chart } from '@/components/chart';
+import { RouteBook } from '@/components/route-book';
+import { Card } from '@penumbra-zone/ui/Card';
+import { useAssets } from '@/shared/state/assets.ts';
+import { useMemo } from 'react';
+
+interface QueryParams {
+  primary: string;
+  numeraire: string;
+}
+
+// Converts symbol to Metadata
+const usePathToMetadata = (params: QueryParams) => {
+  const { data, error, isLoading } = useAssets();
```

Comment:
We should import next-router here to get the params, so that we can use this hook anywhere.